source/_partials/content/composer-updating.html

```diff
@@ -0,0 +1,4 @@
+<div class="alert alert-info" role="alert">
+  <h4 class="info">Note</h4>
+  <p>As packages pulled by Composer are updated (along with their dependencies), version compatibility issues can pop up. `composer.lock` prevents the updating of packages beyond the version limits specified within, and need to be manually reviewed/updated from time to time. See the <a href="https://getcomposer.org/doc/01-basic-usage.md#updating-dependencies-to-their-latest-versions" class="external">updating dependencies</a> section of Composer's documentation for more information.</p>
```

Comment:
composer.lock is meant to stop an version changes unless you updating (not just those beyond the limits in composer.json)
```suggestion
  <p>As packages pulled by Composer are updated (along with their dependencies), version compatibility issues can pop up. Sometimes you may need to manually alter the version constraints on a given package within the `require` or `require-dev` section of `composer.json` in order to update packages. See the <a href="https://getcomposer.org/doc/01-basic-usage.md#updating-dependencies-to-their-latest-versions" class="external">updating dependencies</a> section of Composer's documentation for more information.</p>
```